assemblies/pentaho-solutions/src/main/resources-filtered/pentaho-solutions/system/pentaho.xml

```diff
@@ -164,4 +164,14 @@
   <default-theme>ruby</default-theme>
   <session-expired-dialog>true</session-expired-dialog>
   <set-empty-entity-rest-services>false</set-empty-entity-rest-services>
+    <!--
+    edit permission configuration will be used restrict the content edit permission to user
+    possessing this permission. If you would not like to restrict the users editing the content,
+    simply provide an empty string or remove this configuration from this file all together
+    None = ""
+    Create = org.pentaho.repository.create
+    Publish = org.pentaho.security.publish
+    Admin = org.pentaho.security.administerSecurity
+    -->
+    <edit-permission>org.pentaho.repository.create</edit-permission>
```

Comment:
Will this be enabled by default (breaking), or will it be opt-in?

Reply:
I do see your point, we can definitely keep this as opt-in for the folks wanting to enforce this check. I will change it